browser/brave_profile_prefs.cc

```diff
@@ -24,6 +25,8 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
 
   RegisterAlternatePrivateSearchEngineProfilePrefs(registry);
 
+  RegisterProfilePrefsForBraveThemeType(registry);
```

Comment:
`RegisterProfilePrefsForBraveThemeType` -> `BraveThemeService::RegisterProfilePrefs`

Reply:
Done, all apis in theme_util is moved to static method of `BraveThemeService`.